analytics/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.conf.urls import url
 
-from analytics.views import UserProductivityAnalyticsView
-
+from analytics.views import UserHistoricalAnalyticsView, UserRescueTimeAnalyticsView
 
 urlpatterns = [
-    url(r'^$', UserProductivityAnalyticsView.as_view(), name=UserProductivityAnalyticsView.namespace_url_name),
+    url(r'^$', UserHistoricalAnalyticsView.as_view(), name=UserHistoricalAnalyticsView.namespace_url_name),
+    url(r'^rescuetime/very_productive/correlations/(?P<days_back>[0-9]+)/$',
```

Comment:
You're probably overthinking this since this will have to be refactored for the React calls regardless